examples/hybrid/types.jl

```diff
@@ -126,12 +129,39 @@ function get_spaces(parsed_args, params, comms_ctx)
     dz_top = FT(parsed_args["dz_top"])
     topography = parsed_args["topography"]
 
+    @assert topography in ("NoWarp", "DCMIP200", "Earth")
     if topography == "DCMIP200"
         warp_function = CA.topography_dcmip200
     elseif topography == "NoWarp"
         warp_function = nothing
+    elseif topography == "Earth"
+        data = NCDataset("/groups/esm/asridhar/ETOPO1_Ice_g_gdal.grd")
+        x_range = data["x_range"][:]
+        y_range = data["y_range"][:]
+        z_range = data["z_range"][:]
+        spacing = data["spacing"][:]
+        dimension = data["dimension"][:]
+        elevation = data["z"][:]
+        lon = x_range[1]:spacing[1]:x_range[2]
+        lat = y_range[1]:spacing[2]:y_range[2]
+        nlon = Int64(dimension[1])
+        nlat = Int64(dimension[2])
+        zlevels = reshape(elevation, nlon, nlat)
+        skipind = 50
+        zlevels = zlevels[1:skipind:end, 1:skipind:end]
+        zlevels[zlevels .<= 0.0] .= 0.0
+        zlevels = reverse(zlevels, dims = 2)
+        # Apply Smoothing
+        @info "Apply smoothing kernel..."
+        smooth_degree = 10
+        esmth = imfilter(zlevels, Kernel.gaussian(smooth_degree))
+        step1 = interpolate(esmth, BSpline(Linear()))
+        step2 = scale(step1, lon[1:skipind:end], lat[1:skipind:end])
+        close(data)
+        earth_spline = extrapolate(step2, (Periodic(), Flat()))
```

Comment:
It looks like this extrapolation function is used to obtain elevations of cg grid from the lat/lon grid. Meanwhile the regridding is done by utilize TempestRemap in [ClimaCoupler](https://github.com/CliMA/ClimaCoupler.jl/blob/main/experiments/AMIP/moist_mpi_earth/coupler_utils/regridder.jl). @LenkaNovak may explain why they choose to use TempestRemap rather than a simple extrapolation. It looks fine to me for now but better to use the same method in the end.

Reply:
We use TempestRemap because it ensures preservation of certain properties (namely conservation, consistency and monotonicity) that are important for passing fluxes and states between models. For topography this may not be necessary and a less accurate interpolation should suffice. :)